awx/main/conf.py

```diff
@@ -822,15 +800,6 @@ def _load_default_license_from_file():
 )
 
 
-register(
-    'BROKER_DURABILITY',
```

Comment:
Yep, this was always an experimental thing we tinkered with a year or so ago.  Our hypothesis was "maybe our RabbitMQ will have less performance issues if we optionally disable queue durability."  I don't think we really went anywhere with this, and it's definitely obsolete at this point.

Reply:
https://github.com/ansible/awx/pull/3947